hudi-common/src/main/java/org/apache/hudi/common/config/HoodieStorageConfig.java

```diff
@@ -245,6 +245,14 @@ public class HoodieStorageConfig extends HoodieConfig {
           + "and it is loaded at runtime. This is only required when trying to "
           + "override the existing write context when `hoodie.datasource.write.row.writer.enable=true`.");
 
+  public static final ConfigProperty<String> HOODIE_PARQUET_FLINK_ROW_DATA_WRITE_SUPPORT_CLASS = ConfigProperty
+      .key("hoodie.parquet.flink.rowdata.write.support.class")
+      .defaultValue("org.apache.hudi.io.storage.row.HoodieRowDataParquetWriteSupport")
+      .markAdvanced()
+      .sinceVersion("1.0.1")
```

Comment:
Did you mean 1.1.0? 1.0.1 has been released out.

Reply:
Good catch. It is my mistake.